### PR TITLE
Implement puzzle4 images

### PR DESCRIPTION
**Pull request by iamsammak**
Removed the white background from Chris' images and switched out the numbers for the new images

Also to note: since Chris had 10 images I reformatted the keypad back to a 0-9 instead of 1-9

screen shot 2019-02-16 at 6 10 27 pm

### CODE REVIEW

File: frontend/components/puzzle4.jsx [hunk 1]
```diff
@@ -57,6 +65,19 @@ class Puzzle4 extends React.Component {
     const solved = isSolved(gameState, '4');
     const puzzleBSolved = isSolved(gameState, 'B');
 
+    const images = [
+      { src: 'images/puzzle4/water_droplet.png' },
+      { src: 'images/puzzle4/dog.png' },
```
Comment (ihurrahi) [inline frontend/components/puzzle4.jsx]: are these paths not case sensitive?

File: frontend/components/puzzle4.jsx [hunk 2]
```diff
@@ -1,15 +1,23 @@
 import React from 'react';
+import styled from 'styled-components';
 
 import { Narration } from '../wrappers';
 import { isSolved } from '../helpers';
 import { KeypadContainer, Button, SubmitButton, ClearButton } from './buttonContants';
 import { getCurrentStage } from '../helpers';
 
+const SafeImage = styled.img.attrs(props => ({
+  src: props.src,
```
Comment (ihurrahi) [inline frontend/components/puzzle4.jsx]: dont need this if the prop is just being passed through to the underlying component

File: frontend/components/puzzle4.jsx [hunk 3]
```diff
@@ -70,17 +91,18 @@ class Puzzle4 extends React.Component {
...
-          <Button click={this.state.value[0]} onClick={this.handleClick(0)} >1</Button>
-          <Button click={this.state.value[1]} onClick={this.handleClick(1)} >2</Button>
-          <Button click={this.state.value[2]} onClick={this.handleClick(2)} >3</Button>
-          <Button click={this.state.value[3]} onClick={this.handleClick(3)} >4</Button>
-          <Button click={this.state.value[4]} onClick={this.handleClick(4)} >5</Button>
-          <Button click={this.state.value[5]} onClick={this.handleClick(5)} disabled={puzzleBSolved}>6</Button>
-          <Button click={this.state.value[6]} onClick={this.handleClick(6)} >7</Button>
-          <Button click={this.state.value[7]} onClick={this.handleClick(7)} >8</Button>
-          <Button click={this.state.value[8]} onClick={this.handleClick(8)} >9</Button>
+          <Button click={this.state.value[0]} onClick={this.handleClick(0)} ><SafeImage src={images[0].src} ></SafeImage></Button>
```
Comment (ihurrahi) [inline frontend/components/puzzle4.jsx]: these can be self closing elements (`<SafeImage ... />`)

File: frontend/components/puzzle4.jsx (hunk 2 above)
Comment (iamsammak) [reply frontend/components/puzzle4.jsx]: true statement.

File: frontend/components/puzzle4.jsx (hunk 3 above)
Comment (iamsammak) [reply frontend/components/puzzle4.jsx]: also true statement

File: frontend/components/puzzle4.jsx (hunk 1 above)
Comment (iamsammak) [reply frontend/components/puzzle4.jsx]: that should fix it

Comment (iamsammak) [reply frontend/components/puzzle4.jsx]: fixed

**Pull request merged**